Update CodeQL Style guide to mention acronyms (#5080)
github/codeql: docs/ql-style-guide.md

1. Use [camelCase](https://en.wikipedia.org/wiki/Camel_case) for:
   - Predicate names
   - Variable names

> 1. Abbreviations and acronyms *should* use normal PascalCase/camelCase (if there are only two letters, it is generally ok to use all uppercase letters).

Reviewer: This reads poorly, I think, since the meaning gets mixed up if you read past the open parenthesis ("... *should* use normal PascalCase/camelCase if there are only two letters ...")
Possibly change to something like:
```suggestion
1. Abbreviations and acronyms *should* use normal PascalCase/camelCase (as an exception, if there are only two letters then using all uppercase letters is acceptable).
```

Author: thanks :+1:

  /** ... */
  Type getATypeParameter() { ... }
  

>   /** ... */

Reviewer 2: ```suggestion
  /** Gets the HTTP connection ... */
```

Alignment between the acronym in the QLDoc and in the predicate name is one of the reasons I prefer upper-case acronyms. If we go with Pascal case, at least I'd add this example to show that the acronym should still be all upper case in the QLDoc.

Author: No matter what we end up going with, I think this points about QLDoc is very good :+1: thanks

>   predicate getHttpConnection() { ... }

Reviewer: Possibly add `getSsaVariable`? For me, SSA somehow seems like the canonical example of this.

Author: Done :+1:
